src/app/pages/dashboard/dashboard.tsx

```diff
@@ -42,23 +42,23 @@ export class DashboardPage extends Component {
                         <Card className="balance-card">
                             <CardMediaContent className="balance-card-text">
                                 <p className="mdc-typography--body2">Total Balance</p>
-                                <p className="mdc-typography--headline5">{this.wallet.coin} {this.wallet.balance}</p>
+                                <p className="mdc-typography--headline5">{this.state.coin} {this.state.balance}</p>
                             </CardMediaContent>
                         </Card>
                     </LayoutGrid.Cell>
                     <LayoutGrid.Cell cols={12}>
                         <Card className="address-card">
                             <CardMediaContent className="address-card-text">
                                 <p className="mdc-typography--body2">Wallet address</p>
-                                <p className="mdc-typography--headline5" id="walletAddress">{this.wallet.address}</p>
+                                <p className="mdc-typography--headline5" id="walletAddress">{this.state.address}</p>
```

Comment:
let's try to convert this paragraph in a textarea and use it to copy to clipboard, so no more DOM append and remove needed ;) 

Also for selection of an element from DOM you can use `ref` property. See an example here: https://jsfiddle.net/developit/x8qpzjdf/

Reply:
Ty. I'l take a look now. Also I will delete the bottom bar classes as you suggested and make some squashes for the commits to keep things cleaner